### PR TITLE
Nosereports

### PR DESCRIPTION
**Pull request by techtonik**
Quoting paths to prevent them from failure on paths with quotes, and produce test report in xunit format, which can be then parsed

### CODE REVIEW

**Comment by techtonik** [comment]
This commit was made on the top of #62, which got squashed by Github, so commits from it appeared here again. I guess it is safe to squash them too.

**Comment by jenmud** [comment]
Is there anything that will be using the nosetests xml report ?

File: tox.ini [hunk 1]
```diff
@@ -12,7 +12,8 @@ deps=nose
      assertpy
      pylint
 commands=
-    nosetests -c {toxinidir}/setup.cfg --cover-html-dir={envdir}/coverage --cover-package=ruruki ruruki
+    nosetests -c "{toxinidir}/setup.cfg" --cover-html-dir="{envdir}/coverage" --cover-package=ruruki \
```
Comment (jenmud) [inline tox.ini]: It saddens me that we need to quote this, but oh well I guess it is necessary

**Comment by techtonik** [comment]
Yes, .xml report is needed to make working this tab - https://ci.appveyor.com/project/techtonik/ruruki/build/tests

And also I am thinking that some 3rd party server could be used to upload and track tests metrics from Travis as well, but so far I haven't found any such service.

**Pull request merged**